src/msw/uilocale.cpp

```diff
@@ -96,11 +133,106 @@ wxString wxLocaleIdent::GetName() const
         {
             name << "-" << m_region;
         }
+
+        if (!m_extension.empty())
+        {
+            name << "-" << m_extension;
+        }
+
+        if (!m_sortorder.empty())
+        {
+            name << "_" << m_sortorder;
+        }
     }
 
     return name;
 }
 
+// ----------------------------------------------------------------------------
+// Standard C wxUILocale implementation for MSW
```

Comment:
It would be nice to explain why do we need a separate class for this instead of just using "en-US" locale as before, as this is not obvious to me.

Reply:
At first, I used locale _"en-US"_. However, the standard C locale and the locale _"en-US"_ are not identical - if I remember correctly some standard date formats do not match (at least in the GLIBC implementation). Therefore some tests failed, and I decided to use a dedicated implementation for the C locale to be on the safe side.